src/Symfony/Component/ClassLoader/ApcUniversalClassLoader.php

```diff
@@ -60,7 +60,7 @@
  *
  * @api
  */
-class ApcUniversalClassLoader extends UniversalClassLoader
+class ApcUniversalClassLoader extends ClassLoader
```

Comment:
Thanks for the notice! I will update the PR. The `ApcUniversalClassLoader` is not tagged as `@deprecated` anywhere. That's why I thought we had to keep it. I will update my deprecation notices PR as well to mark the class deprecated.